sql-db/hibernate-fulltext-search/src/main/java/io/quarkus/qe/hibernate/search/Fruit.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.qe.hibernate.search;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.xml.bind.annotation.XmlRootElement;
+
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.search.engine.backend.types.Sortable;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.FullTextField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.KeywordField;
+
+@Entity
+@Table(name = "known_fruits")
+@NamedQuery(name = "Fruits.findAll", query = "SELECT f FROM Fruit f ORDER BY f.name")
+@NamedQuery(name = "Fruits.findByName", query = "SELECT f FROM Fruit f WHERE f.name=:name")
+@XmlRootElement(name = "fruit")
+@Indexed
+public class Fruit {
+
+    @Id
+    @GeneratedValue(generator = "fruitsSequence")
+    @GenericGenerator(name = "fruitsSequence", strategy = "io.quarkus.qe.hibernate.search.MyCustomIdGenerator")
```

Comment:
I have added a custom ID generator because "sequenceGenerator" used by Postgres is not compatible with Mysql. So, in order to re-use the scenario, I decided to implement this generator. Also by this way, we are testing also this feature.